README.md

```diff
@@ -0,0 +1,184 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[ Budget App\] ](#--budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [Author](#author)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+# 📖 [ Budget App] <a name="about-project"></a>
+
+> Description the project.
+
+1. Install Ruby on Rails framework.
+2. Use Ruby Gems as a software packages system.
+3. Understand what ORM is.
+4. Use database migration files to maintain database schema.
+5. Write tests for models, controllers, system.
+5. Set up associations between models.
+6. Write SQL queries with Active Record
+7. Use preprocessed HTML file with embedded Ruby code.
+8. Use layouts and templates for shared content.
+9. Build a web app that requires the user to log in.
+10. Use devise gem for authentication.
+11. Limit access to web app resources based on authorization rules.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+ 
+
+- **Create a budget app**
+- **register and log in, so that the data is private to them.** 
+- **introduce new transactions associated with a category.**
+- **see the money spent on each category.**
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+
+- [Live Demo Link]()
+
```

Comment:
- [ ] [REQUIRED] Please you should be deploy your application using `Heroku`, `Netlify` or `GitHub Pages` because the project should deployed properly, the `home` and `detail` page works as expected 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/221377521-ac021695-de0e-4ba6-b0ca-8154ab0e44e1.png)

Reply:
My project was deployed https://budget-app-bmmv.onrender.com/

Reply:
> My project was deployed https://budget-app-bmmv.onrender.com/

@ecekpo Nice, please add this link to your README which required place 👍🏼